lib/paymentrails/gateways/RecipientGateway.rb

```diff
@@ -29,6 +29,25 @@ def delete(recipient_id)
       true
     end
 
+    def delete_multiple(recipient_ids)
+      @client.delete_multiple('/v1/recipients/', ids: recipient_ids)
+      true
+    end
+
+    def retrieve_logs(recipient_id)
+      response = @client.get('/v1/recipients/' + recipient_id + '/logs')
+      JSON.parse(response, object_class: OpenStruct)
```

Comment:
There's no `Log` class, I think this is fine though; similar to the approach in 

https://github.com/PaymentRails/ruby-sdk/blob/974fa5245c3078acf20ee8044da76b3fdd7a224f/lib/paymentrails/gateways/BalanceGateway.rb#L9-L11

Reply:
@zero-bites You mean there's no Log class representing the APIs for logs?

Reply:
That's right, we're missing modeling for both `Log` and `Balance`, should we fix that?

Reply:
Will this change the behavior of what the callers of this method receive, and hence we'll need to deprecate the existing method and introduce a new one?

If this doesn't change scope, let's do this and note it in our change log?

Reply:
I can add a model for Logs, but don't think adding a model for Balance is good because it's already in prod. 

Will add a model to represend Logs

Reply:
I've changed my mind, DM'd you